users/tests/test_user.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from django.test import Client
+from django.contrib.auth.models import User
+
+
+@pytest.fixture
+def user():
+    return User.objects.create_user('Test-user', 'Test-user@mail.com', 'Password')
+
+
+@pytest.mark.django_db
+def test_user_creation(user):
+    assert User.objects.filter(pk=user.id).exists()
+    assert User.objects.get(pk=user.id) == user
+
```

Comment:
'user' fixture is saving the user to the DB.
'assert User.objects.get(pk=user.id) == user' - here you are checking if the function get is working.
 I think you should change 'user' fixture to return an object that didn't save to the DB and then,
 you can save the user in  'test_user_creation' and add the assertation you did. that way you check if the user saved correctly to the DB.